app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -101,6 +101,11 @@
         return Lists.newArrayList("REDISCLOUD_URL", "REDISTOGO_URL");
     }
     
+    @Bean(name = "newRedisProviders")
+    public List<String> newRedisProviders() {
+        return Lists.newArrayList(BridgeConfigFactory.getConfig().get("elasticache.url"));
+    }
```

Comment:
redisProviders() returns the env var names that we lookup to get the Redis URL. However, newRedisProviders() seems to be getting the value of elasticache.url. Is this intentional? Is that value supposed to be another env var name? Or a Redis URL?

Also, given that we have no plans to use REDISTOGO_URL anytime soon, is it worth getting rid of these altogether and just have fixed env var names? This might simplify the configs.